README.md

```diff
@@ -11,11 +11,27 @@ Pre-requisites: `python`, `pip`, `git`, `bazel`
 To do a test run:
 
 1.  The use of `virtualenv` is strongly recommended. Do this before you carry on
```

Comment:
I didn't do all these virtualenv steps. The build still succeeded. Is that the reason for my test failures?